### PR TITLE
fixed:[doc] resize the search icon in search bar #5099

### LINKED ISSUE #5099: [doc] resize the search icon in search bar
### Summary

Currently, the search icon on the search bar of the documentation is a bit large.
Update the search icon size to `13x13`

Screenshot 2022-12-28 at 1 42 26 PM

### Issue Type

Documentation feature/update

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs

### Steps to reproduce the issue

1. Open Docs
2. Check the search bar on top-left

### Additional Information

_No response_

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by sudip57**
Fixes #5099 
The size of the search icon was 20 px as it has mentioned i changed into 13 x 13

### CODE REVIEW

**Review by withshubh — CHANGES_REQUESTED** [review]
Please remove all the unnecessary changes from this PR and just keep the change that resizes the search icon.

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -257,18 +263,25 @@ strong {
...
 }
 
 .DocSearch-Button {
   border-radius: 5px !important;
 }
 
+.DocSearch-Search-Icon {
+  height: 13px;
+  width: 13px;
+}
```
Comment (withshubh) [inline docs/src/css/custom.css:280]: Only this change is required. Please remove all other changes from this PR. (Formatting and other changes)

Comment (sudip57) [reply docs/src/css/custom.css:272]: I have removed the unnecessary changes.

**Comment by sudip57** [comment]
Made the changes

**Review by withshubh — APPROVED** [review]

**Pull request merged**